tests/testthat/test-combinevariablesetsasbinary.R

```diff
@@ -0,0 +1,111 @@
+library(testthat)
```

Comment:
This new file seems to be using tabs instead of spaces for indentiation in lots of places.